### ISSUE TITLE
JSReport poblem creating a PDF report local server.

### ISSUE THREAD

**Issue by coltcorrea** [jsreport/jsreport#398]
I am trying to create a PDF report using JSReport on a local Windows 7 Machine.  The server crashes and leaves no error.  Can someone help me fix this issue?

http://localhost:3004/api/quote/driveCallback
2018-03-29T17:00:43.828Z - info: Initializing jsreport@1.9.2 in development mode using configura
tion file: none
2018-03-29T17:00:43.830Z - info: Setting process based strategy for rendering. Please visit http
://jsreport.net/learn/configuration for information how to get more performance.
2018-03-29T17:00:43.835Z - info: Searching for available extensions in E:\blusky\BlueSky\
2018-03-29T17:00:43.840Z - info: Extensions location cache not found, crawling directories
2018-03-29T17:00:45.294Z - info: Found 30 extensions
2018-03-29T17:00:45.327Z - debug: Writing extension locations cache to C:\Users\ccorrea\AppData\
Local\Temp\jsreport\core\locations.json
2018-03-29T17:00:45.330Z - debug: Discovered 30 extensions
2018-03-29T17:00:45.333Z - info: Using extension jsrender
2018-03-29T17:00:45.335Z - info: Using extension fop-pdf
2018-03-29T17:00:45.405Z - info: Using extension handlebars
2018-03-29T17:00:45.408Z - info: Using extension templates
2018-03-29T17:00:45.421Z - info: Using extension html-to-xlsx
2018-03-29T17:00:45.527Z - info: Using extension data
2018-03-29T17:00:45.529Z - info: Using extension phantom-pdf
2018-03-29T17:00:45.599Z - info: Using extension text
2018-03-29T17:00:45.601Z - info: Using extension assets
2018-03-29T17:00:45.635Z - info: Using extension scripts
2018-03-29T17:00:45.642Z - info: Using extension xlsx
2018-03-29T17:00:46.137Z - info: Creating memory store
2018-03-29T17:00:46.166Z - info: reporter initialized
2018-03-29T17:00:46.167Z - info: Starting rendering request 1 (user: null) requestId=1
2018-03-29T17:00:46.169Z - info: Rendering anonymous template { recipe:phantom-pdf,engine:handle
bars} requestId=1
2018-03-29T17:00:46.169Z - debug: Data item not defined for this template. requestId=1
2018-03-29T17:00:46.176Z - debug: Rendering engine handlebars requestId=1
2018-03-29T17:00:46.564Z - debug: Compiled template not found in the cache, compiling timestamp=
Thu Mar 29 2018 13:00:46 GMT-0400 (Eastern Daylight Time), requestId=1
2018-03-29T17:00:46.567Z - debug: Executing recipe phantom-pdf requestId=1

ccorrea@Trivium MINGW64 /e/blusky/BlueSky (master)

**Comment by pofider** [comment]
Please provide the full request specification. Does this happen only for specific report or always? Thank you.

**Comment by coltcorrea** [comment]
Hello and thank you for your response.  The crash was happening on ALL reports regardless of the content.  It turns out that something was something with "node_modules".  To fix this problem, here is what I did.  Hopefully this helps others:

1. Delete the "node_modules"
2. run "npm install"

Before I ran "npm install" many times and this did not fix the issue but deleting "node_modules" first then "npm install" fixed the issue for me.
